### PR TITLE
WIP. add py-pyyaml@5.3.1, py-morph-tool@0.2.10

### PR DESCRIPTION
**Pull request by asanin-epfl**

### CODE REVIEW

**Comment by asanin-epfl** [comment]
- For `py-pyyaml@5.3.1` I had to download its tar.gz, and get sha256 from it manually. Is that correct? This package is not on pypi.
- For plotly I switched to pypi releases otherwise I had to follow the path of `py-pyyaml` as above.

**Comment by matz-e** [comment]
>     * For `py-pyyaml@5.3.1` I had to download its tar.gz, and get sha256 from it manually. Is that correct? This package is not on pypi.

You should be able to do `spack checksum py-pyyaml`, too.

**Comment by asanin-epfl** [comment]
thank you @matz-e . I've checked with `spack checksum` and it gave the same sha256. I keep receiving this error at [CI](https://bbpcode.epfl.ch/ci/job/hpc.spack-deployment.pull-requests/1129/testReport/junit/(root)/py-ipykernel/khlk52nuqv6es5ndxmz6hfkosw2qbvjj/):
```
caxzuyno4x/lib/python3.7/site-packages/IPython/core/events.py", line 16, in <module>
    from backcall import callback_prototype
ModuleNotFoundError: No module named 'backcall'
```
However this module is present. May be you saw this before because I'm puzzled how to fix it.

**Comment by asanin-epfl** [comment]
retest this please Jenkins

**Comment by asanin-epfl** [comment]
And what is the problem now? I don't see anything in the logs by searching with 'error' of 'fail' keyword. Just a general failure mark in the end of logs without any explanation.

**Comment by matz-e** [comment]
> And what is the problem now? I don't see anything in the logs by searching with 'error' of 'fail' keyword. Just a general failure mark in the end of logs without any explanation.

In general, it's way easier to use BleuOcean. So… after clicking on "Details", following up with BleuOcean, I see that the subsequent `hpc.module-testing` failed. Note the build number, look in BleuOcean for that plan and build number. In your case, I see that the failure is:
```
ImportError: /gpfs/bbp.cscs.ch/ssd/apps/hpc/jenkins/deploy/applications/2020-02-01/linux-rhel7-x86_64/intel-19.0.4/py-scipy-1.3.1-3lmmoqxl7d/lib/python3.7/site-packages/scipy/linalg/_fblas.cpython-37m-x86_64-linux-gnu.so: undefined symbol: __intel_sse2_strcat

script returned exit code 1
```
Which looks like a thing that occurs with `%intel`… @pramodk or @jorblancoa might have seen and solved this before, I've seen it, too. Need to dig around a little.

**Comment by asanin-epfl** [comment]
Thank you for explanation! Next time I will try to deal with such errors on my own.

**Comment by asanin-epfl** [comment]
@matz-e or @pramodk could please advice on this problem? I have no idea what to do.

File: var/spack/repos/builtin/packages/py-plotly/package.py [hunk 1]
```diff
@@ -10,10 +10,11 @@ class PyPlotly(PythonPackage):
...
 
     homepage = "https://plot.ly/python/"
-    url      = "https://github.com/plotly/plotly.py/archive/v2.2.0.tar.gz"
+    url      = "https://pypi.io/packages/source/p/plotly/plotly-3.0.0.tar.gz"
 
-    version('3.0.0', sha256='aa8e0020e139f1a3413df578ec3e3162c6ccb2d4c6d66f6fd98547c3ff83ce0b')
-    version('2.2.0', sha256='dad2a49fe355dddb6ae159e96c10ac22413a33cbac513b4dbf3791e63ec33c1f')
+    version('4.1.0', sha256='9333f28feacaa0751df04cccfb266463bf5029dcb106d3f81d7af5942fd3d998')
+    version('3.0.0', sha256='baab7f8d54f37df89f31d6e6e5d2d4b491bccb79e5d490e25d5762bb762a277c')
+    version('2.2.0', sha256='ca668911ffb4d11fed6d7fbb12236f8ecc6a7209db192326bcb64bdb41451a58')
```
Comment (matz-e) [inline var/spack/repos/builtin/packages/py-plotly/package.py:17]: Seems like there are more dependencies here:
```
pkg_resources.DistributionNotFound: The 'retrying>=1.3.3' distribution was not found and is required by plotly
```

**Review by matz-e — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/py-plotly/package.py (hunk 1 above)
Comment (asanin-epfl) [reply var/spack/repos/builtin/packages/py-plotly/package.py:17]: Thank you! it helped. I also rebased the PR on the latest `develop`. Manually checked that everything works via `source /gpfs/bbp.cscs.ch/apps/hpc/jenkins/pulls/930/config/modules.sh`.

**Review by matz-e — APPROVED** [review]

**Comment by asanin-epfl** [comment]
Yay, thank you!

**Pull request merged**